addons/udes_stock/models/stock_inventory.py

```diff
@@ -58,6 +58,11 @@ def _get_stock_check_users(self):
         domain=_get_stock_check_users,
     )
 
+    @api.onchange("filter")
+    def onchange_filter(self):
+        self.exhausted = False
+        super().onchange_filter()
```

Comment:
Will the super call not set `exhausted` back to `True` potentially?

Reply:
Ah okay I think I get it, Odoo only sets exhausted to `True` if filtering is set to `product`, and that will only ever be 1 product right?

Reply:
In fact i was thinking same thing, maybe safer to set exhausted to False after super.

Reply:
Yep, only one product line will be created.

Reply:
@armandcela I think we should leave it as it is. Because if we set `exhausted` after the `super` call, then it means that specifying one product only will not add a line by default in the Inventory Adjustment if it doesn't have stock in the location.